microsoft/testsuites/network/sriov.py

```diff
@@ -399,23 +685,33 @@ def _sriov_vf_connection_test(
                 expected_exit_code_failure_message="Fail to copy file large_file from"
                 f" {source_ip} to {dest_ip}",
             )
-
+            source_tx_packets = self._get_packets(source_node, source_nic)
+            dest_tx_packets = self._get_packets(dest_node, dest_nic, "rx_packets")
             # verify tx_packets value of source nic is increased after coping 200Mb file
             #  from source to dest
-            source_tx_packets = source_cat.read_from_file(
-                f"/sys/class/net/{source_nic}/statistics/tx_packets", force_run=True
-            )
             assert_that(
                 int(source_tx_packets), "insufficient TX packets sent"
             ).is_greater_than(int(source_tx_packets_origin))
             # verify rx_packets value of dest nic is increased after receiving 200Mb
             #  file from source to dest
-            dest_tx_packets = dest_cat.read_from_file(
-                f"/sys/class/net/{dest_nic}/statistics/rx_packets", force_run=True
-            )
             assert_that(
                 int(dest_tx_packets), "insufficient RX packets received"
             ).is_greater_than(int(dest_tx_packets_origin))
+
             if turn_off_vf:
                 source_node.execute(f"ip link set dev {source_vf_nic} up", sudo=True)
                 dest_node.execute(f"ip link set dev {dest_vf_nic} up", sudo=True)
+
+    def _stop_firewall(self, environment: Environment) -> None:
+        for node in environment.nodes.list():
+            firewall = node.tools[Firewall]
+            firewall.stop()
+
+    def _kill_iperf3(self, environment: Environment) -> None:
```

Comment:
This method is used in after_case once, and it's short. It doesn't need to be a method now.

Reply:
update